docs-ref-conceptual/azure-cli-endpoints.md

```diff
@@ -0,0 +1,220 @@
+--- 
+title: Endpoints used when installing the Azure CLI  | Microsoft Docs
+description: Learn what endpoints are used when installing the Azure CLI.  These URLs are sometimes added to allowlists when working behind a firewall.
+manager: mkluck
+author: dbradish-microsoft
+ms.author: dbradish
+ms.prod: non-product-specific
+ms.topic: conceptual
+ms.date: 11/14/2022
+ms.tool: azure-cli
+---
+# Overview
+If your organization is secured with a firewall or proxy server, you must add certain internet protocol (IP) addresses and domain uniform resource locators (URLs) to the **allowlist** prior to installing the Azure CLI.
```

Comment:
```suggestion
If your organization is secured with a firewall or proxy server, you must add certain IP (internet protocol) addresses and domain URLs (uniform resource locators) to the **allowlist** prior to installing the Azure CLI.
```

I'm not sure if this is a great format and we can ignore if you disagree, it's just that the acronyms are really the well known monikers and to me, should be used as the primary.